procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,87 @@
+# Council Synchronous Meetings
+
+This documents procedures, norms, and logistics of the Council's sync meetings.
+
+## Frequency
+
+The Council meets once a week.
+
+On the first meeting on or after the 14th of July, 2023, the Council will re-evaluate whether this frequency still makes sense or whether it should be changed.
+
+## Location
+
+The meeting happens in some video call software (e.g., Google Meet).
+
+## Quorum
+
+The meeting has reached quorum if at *most* 2 Council representatives (and their alternates) are not present (i.e., N - 2).
+
+If quorum is not reached, then the sync meeting is canceled and the agenda is gone through asynchronously in chat.
+
+## Minutes
+
+Detailed minutes must be taken of the meeting even if the meeting was otherwise recorded. The meeting minutes must be made public within one week after the meeting concludes (or a delay must be publicly announced and resolved as soon as possible). After the meeting, Council members must review the minutes to ensure that the minutes reflect their understanding of what occurred in the meeting. Once consensus is reached that the minutes are accurate, they are released for public viewing.
+
+Minutes may be found [in the 'minutes' directory of the leadership Council team repository](../minutes).
+
+## Recording
+
+All meetings are recorded, but consensus must be reached after the meeting on what do with the recording. The Council has no obligation to do anything with the recording. If after one week, consensus on what to do with the recording has not been reached, the default is to delete the recording. 
+
+Recordings are usually used to ensure that the notes are correct and to optionally create automatic transcripts. Recordings can also *optionally* be made public, but only with consensus of all of those present on the recording.
+
+## Meeting roles
+
+The meeting has the following assigned roles:
+
+- **Facilitator**: the facilitator is in charge of pushing discussion along, calling on hands, formally asking the group whether there are any objections to a particular motion, and keeping the discussion on agenda.
+- **Scribe(s)**: the scribes are in charge of taking notes on what was discussed in the meeting including noting any action items or follow up questions.
+
+## Meeting Agenda
+
+The facilitator is in charge of ensuring that a proposal for the main agenda for the meeting is made publicly available as a pull request (as the start of the documented minutes for the meeting) to the Council team repository at least 2 days before the meeting. All Council members may comment on, revise, add to or otherwise collaborate on ensuring the agenda is in the right shape.
```

Comment:
This seems to imply that "Facilitator" is a standing role, but then the role is assigned during the metings.
I have resolved such in my groups by having two roles "Chair" and "Speaker" - the Chair being a standing role and the *de facto* default speaker, and handling preparation o the agenda and minutes, and the Speaker being chosen during the meeting to run that meeting. Prior to a Speeker being chosen, the Chair conducts the inital items if present otherwise if they expect to be Absent, they can appoint an acting chair to conduct as much of the Chair's duties as necessary (else, the council could appoint them, perhaps synchronously).

Reply:
This is a good suggestion. I'm currently trying to document the status quo, but I like this suggestion and will see I can incorporate it into how we do meetings. Currently I've been playing the role of both Chair and Speaker in this scenario.